namespacesmapper/namespacesmapper.go

```diff
@@ -0,0 +1,317 @@
+package namespacesmapper
```

Comment:
Since this file already gets pretty long and complex would it make sense to split it into two files - one for dynakubes, one for namespaces?